content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 1. Integer
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Integer.OI
+  - ib-19.3.Datatype-Primitief-Integer.OI
+---
+
+## Wat is een Integer?
+Een Integer is een datatype waar een positief of negatief getal in past. Een programmeur kan dit datatype aan een variabel geven en hiermee getallen bewaren, bewerken en uitlezen.
```

Comment:
Missend: bij het conceptueel model (dit deel) wordt een aanleiding verwacht (as in: waarom is dit ding nodig?)